eb_sqs/auto_tasks/service.py

```diff
@@ -24,7 +24,8 @@ def _auto_task_wrapper(module_name, class_name, func_name, *args, **kwargs):
         class_ = getattr(module, class_name)  # find class
 
         auto_task_executor_service = _AutoTaskExecutorService(func_name)
-        instance = class_(auto_task_service=auto_task_executor_service)  # instantiate class using _AutoTaskExecutorService
+        instance = class_(
+            auto_task_service=auto_task_executor_service)  # instantiate class using _AutoTaskExecutorService
```

Comment:
nit: indentation - closing bracket and comment could be on next line (and comment can probably be removed altogether)

just wanted to add a comment on this PR! 😁

Reply:
I thought GitHub was messing with me when I saw the name.. 😁 Good to see you watching...